### PR TITLE
generate random room name

### LINKED ISSUE #14: Generate random room name button in room name dialoug
Problem:
Users often get confused to choose a name for their room, that name could also be already used by someone

Solution:
Generate a random name for the room, it could be a random thing thats readable: `AmazingGooseMan` or something unreadable like an id/hash/random string of text

Tasks:
- [ ] this is where u add smaller broken down tasks

### PR DESCRIPTION
**Pull request by RandSohail**
Create a small function to generate a random room name 
Screenshot from 2023-05-01 09-59-25

### CODE REVIEW

File: client/src/features/Home/RoomNameForm.tsx [hunk 1]
```diff
@@ -77,16 +84,28 @@ export const RoomNameForm: React.FC<Props> = ({ open, handleClose, youtubeUrl })
...
+                  autoFocus
+                />
+                <Button
+                  onClick={generateRandomRoomName}
+                  sx={{ mt: 2 }}
+                  variant="contained"
+                >
+                  Random Name
+                </Button>
+              </>
```
Comment (Amoodaa) [inline client/src/features/Home/RoomNameForm.tsx:106]: Can u add a screenshot of how this looks like in the form? in the description
Also maybe a text field adornment https://mui.com/material-ui/react-text-field/#input-adornments might look better

Comment (RandSohail) [reply client/src/features/Home/RoomNameForm.tsx:106]: I add a screenshot so you can see what I've done.
I think if I use text field adornment there won't be an icon to symbole ( generate name ), I think a button will be the best choice.

**Comment by Jood80** [comment]
@RandSohail Styling suggestion:
Have you tried putting the `Generate Name` button next to the left of `CREATE ROOM` button and see how it looks there? 
Also, you maybe want to capitalize your button name too to match its neighbor :)

File: client/src/features/Home/RoomNameForm.tsx (hunk 1 above)
Comment (Amoodaa) [reply client/src/features/Home/RoomNameForm.tsx]: okay fair points, but you can use it with a combination of a tooltip, im not a designer im good with this

**Review by Amoodaa — APPROVED** [review]

**Pull request merged**